### ISSUE TITLE
install error

### ISSUE THREAD

**Issue by bsb2014** [Gaius-Augustus/Augustus#399]
command is 
singularity build augustus.sif Singularity.def
FATAL:   You must be the root user, however you can use --remote or --fakeroot to build from a Singularity recipe file

Does the singularity require the root to install augustus? 

Thanks.

**Comment by KatharinaHoff** [comment]
No, you should not need root permissions. However, it may be version dependent. We tested with singularity-ce version 3.11.3

**Comment by bsb2014** [comment]
Is there a 3.11.3 link to download? thanks

**Comment by StepanSaenko** [comment]
> Is there a link to download 3.11.3? thanks

Here, on Github: https://github.com/sylabs/singularity/releases/tag/v3.11.3
